Mail/Views/Thread/ThreadView.swift

```diff
@@ -86,8 +87,12 @@ struct ThreadView: View {
 
                 LazyVStack(spacing: 0) {
                     ForEach(messages.indices, id: \.self) { index in
-                        let isMessageExpanded = ((index == messages.count - 1) && !messages[index].isDraft) || !messages[index].seen
+                        let isMessageExpanded = ((index == messages.count - 1) && !messages[index].isDraft) || !messages[index]
+                            .seen
                         MessageView(message: messages[index], isMessageExpanded: isMessageExpanded)
+                            .modifyIf(messages.count > 1) {
```

Comment:
I added the `modifyIf` modifier to the project several months ago, when my SwiftUI knowledge was limited, we should avoid it.

Whats does this modifier do :
```
if condition {
    view.padding()
else {
    view
}
```
In result, our ThreadView contains two MessageView in its view hierarchy and the view becomes :
`ModifyContent<MessageView, MessageView>`